### ISSUE TITLE
New config option `withoutValidators`

### ISSUE THREAD

**Issue by nodkz** [apollographql/eslint-plugin-graphql#98]
I want to use all current list of validators for Relay, except `NoUnusedVariables` validator.

For now, I can describe all validators explicitly but may lose some new validators in future.
Guys, what do you think about new option `withoutValidators`?

**Comment by jnwng** [comment]
i have to think more carefully about how to weigh the difference between the existing concept of an "environment" (which maps to a [subset of all GraphQL validators](https://github.com/apollographql/eslint-plugin-graphql/blob/585288bf02773d1c80a4adbd4d637e2139369b82/src/index.js#L26-L54)) vs being able to control the individual validators themselves.

as you've mentioned, perhaps this looks like combining the validators set by the environment, and adding and subtracting them based on an includes/excludes validators and utilizing the result

**Comment by jayqui-omada** [comment]
Would find this feature useful. `excludeValidators` strikes me as a somewhat more intuitive name.

**Comment by Nnoerregaard** [comment]
I'd really love this feature too! Is there any workaround at the moment other than explicitly defining all validators except the few I want to exclude?

**Comment by Abuzhan** [comment]
+ 1 for me. Would be very useful!

**Comment by oneserv-heuser** [comment]
This would be an awesome feature!

**Comment by binishkaspar** [comment]
Yes. This would be a nice feature to have. However I would suggest something like this,

```js
{
   "validators": {
        "NoUnusedFragments": false,
        "NoUnusedVariables": { "enabled": true, ...someConfigs },
   }
}
```
